### PR TITLE
Slurm validation

### PR DESCRIPTION
**Pull request by priteau**

### CODE REVIEW

File: playbooks/validation.yml [hunk 1]
```diff
@@ -51,6 +46,9 @@
 
 # run some simple slurm jobs
 - hosts: login
+  become: true
+  become_user: "{{ jasmin_users | map(attribute='username') | first }}"
+  become_flags: "--login" # Use a login shell to load the module function and ensure $HOME is set
```
Comment (oneswig) [inline playbooks/validation.yml:51]: Nice

File: playbooks/validation.yml [hunk 2]
```diff
@@ -51,6 +46,9 @@
 
 # run some simple slurm jobs
 - hosts: login
+  become: true
+  become_user: "{{ jasmin_users | map(attribute='username') | first }}"
```
Comment (oneswig) [inline playbooks/validation.yml:50]: I hope the first user doesn't mind being used, these jobs might be accounted for somewhere :-)

**Review by oneswig — APPROVED** [review]
Looks good, nice to see this work continue.

File: playbooks/validation.yml (hunk 2 above)
Comment (priteau) [reply playbooks/validation.yml:50]: I suppose we'll want to run the validation playbook only on short-lived test clusters.

**Pull request merged**